stablehlo/reference/ProcessGrid.h

```diff
@@ -66,6 +68,9 @@ class RendezvousResult {
   /// `processId`. If key is not found, return an empty `Tensor`.
   Tensor lookup(ProcessId processId);
 
+  /// Returns the tensor values of the map `result_`.
```

Comment:
Let's write this comment in terms of the public API. I don't think that users of RendezvousResult should know about `result_`.